presto-hive/src/test/java/com/facebook/presto/hive/TestHiveFileFormats.java

```diff
@@ -785,6 +786,15 @@ private FileFormatAssertion assertThatFileFormat(HiveStorageFormat hiveStorageFo
                 .withStorageFormat(hiveStorageFormat);
     }
 
+    private static HiveClientConfig getParquetHiveClientConfig(boolean optimizedReader, boolean usePushDown, boolean useParquetColumnNames)
```

Comment:
To be consistent with the methods names in `HiveClientConfig`:
- `optimizedReader` -> `enableOptimizedReader`
- `usePushDown` -> `enablePredicatePushDown`

Reply:
You can chain calls:

```
HiveClientConfig config = new HiveClientConfig();
config.setParquetOptimizedReaderEnabled(optimizedReader)
        .setParquetPredicatePushdownEnabled(usePushDown)
        .setUseParquetColumnNames(useParquetColumnNames);
return config;
```